### PR TITLE
Adds Post Upgrade Existance tests for location and ostree-branch

### PR DESCRIPTION
**Pull request by jameerpathan111**
Adds Post Upgrade Existance tests for location and ostree-branch issue #32

### CODE REVIEW

File: upgrade_tests/test_existance_relations/cli/test_location.py [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+:CaseLevel: System
+
+:CaseComponent: CLI
+
+:TestType: nonfunctional
+
+:CaseImportance: High
+
+:SubType1: installability
```
Comment (jyejare) [inline upgrade_tests/test_existance_relations/cli/test_location.py]: remove this not required.

File: upgrade_tests/test_existance_relations/cli/test_location.py [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+loc_desc = compare_postupgrade(component, 'description')
+loc_title = compare_postupgrade(component, 'title')
+
+
+# Tests
+@pytest.mark.parametrize("pre,post", loc_id, ids=pytest_ids(loc_id))
+def test_positive_locations_by_id(pre, post):
+    """Test all locations are existing after upgrade by id's
+
+    :id: upgrade-d16b33af-46b3-4413-b018-e7a4926c8bcc
```
Comment (jyejare) [inline upgrade_tests/test_existance_relations/cli/test_location.py]: make sure these are all new, ignore if it is already

File: upgrade_tests/test_existance_relations/cli/test_ostree_branch.py [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+
+:CaseLevel: System
+
+:CaseComponent: CLI
+
+:TestType: nonfunctional
+
+:CaseImportance: High
+
+:SubType1: installability
```
Comment (jyejare) [inline upgrade_tests/test_existance_relations/cli/test_ostree_branch.py]: same as above

File: upgrade_tests/test_existance_relations/cli/test_ostree_branch.py [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+    :id: upgrade-057ba0be-581c-4892-ad99-c52f0c3fd7c9
+
+    :expectedresults: All ostree-branch should be retained post upgrade by id's
+    """
+    assert existence(pre, post)
+
+
+@pytest.mark.parametrize("pre,post", ostree_name, ids=pytest_ids(ostree_name))
+def test_positive_ostree_branch_by_name(pre, post):
+    """Test all ostree-branch are existing after upgrade
```
Comment (jyejare) [inline upgrade_tests/test_existance_relations/cli/test_ostree_branch.py]:
```suggestion
    """Test all ostree-branch are existing after upgrade by name
```

File: upgrade_tests/test_existance_relations/cli/test_ostree_branch.py [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+
+    :id: upgrade-9e8e74ba-43f7-41e5-ae5e-2b745f2a76fe
+
+    :expectedresults: All ostree-branch should be retained post upgrade
```
Comment (jyejare) [inline upgrade_tests/test_existance_relations/cli/test_ostree_branch.py]:
```suggestion
    :expectedresults: All ostree-branch should be retained post upgrade by name
```

File: upgrade_tests/test_existance_relations/cli/test_ostree_branch.py [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+    """
+    assert existence(pre, post)
+
+
+@pytest.mark.parametrize("pre,post", ostree_version, ids=pytest_ids(ostree_version))
+def test_positive_ostree_branch_by_version(pre, post):
+    """Test all ostree-branch are existing after upgrade
```
Comment (jyejare) [inline upgrade_tests/test_existance_relations/cli/test_ostree_branch.py]:
```suggestion
    """Test all ostree-branch versions are existing after upgrade
```

File: upgrade_tests/test_existance_relations/cli/test_ostree_branch.py [hunk 7]
```diff
@@ -0,0 +1,62 @@
...
+
+    :id: upgrade-9dc8583e-b7fb-43b3-81d4-5ae0f2917ff8
+
+    :expectedresults: All ostree-branch should be retained post upgrade
```
Comment (jyejare) [inline upgrade_tests/test_existance_relations/cli/test_ostree_branch.py]:
```suggestion
    :expectedresults: All ostree-branch versions should be retained post upgrade
```

**Comment by jameerpathan111** [comment]
thanks @jyejare for comments, will update PR accordingly

File: upgrade_tests/test_existance_relations/cli/test_location.py (hunk 2 above)
Comment (jameerpathan111) [reply upgrade_tests/test_existance_relations/cli/test_location.py]: yes, i have added new test id's for all tests.

**Comment by jameerpathan111** [comment]
@jyejare I have updated the PR.

**Comment by san7ket** [comment]
Test results Please

**Comment by jyejare** [comment]
@jameerpathan111 , Can we get the test results for this ?

**Comment by jyejare** [comment]
whats up @jameerpathan111  ? :)

**Comment by jyejare** [comment]
@jameerpathan111 Try to retest and merge this with the help of @devendra104 or @jyejare .

**Comment by jameerpathan111** [comment]
> @jameerpathan111 Try to retest and merge this with the help of @devendra104 or @jyejare .

ok :)

**Comment by devendra104** [comment]
This is long pending PR, if you are not planning to do anything around it then please close.

**Comment by jameerpathan111** [comment]
I am not planing to work on this soon, so closing this.

**Pull request closed without merging**